src/main/resources/db/migration/V5.11.0.2__hdc-3b2-template-instructions.sql

```diff
@@ -0,0 +1,14 @@
+ALTER TABLE template ADD COLUMN IF NOT EXISTS instructions TEXT;
```

Comment:
... and, per a dv-tech slack thread, there's a PR from an outside developer already in QA (https://github.com/IQSS/dataverse/pull/8793) that has a flyway script numbered V5.11.0.2.

Reply:
updated